<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="quad" default="SQ01s"/>

    <node ns="$(arg quad)" name="obstacle" pkg="rmader" type="obstacle.py" output="screen" clear_params="true">
        <rosparam file="$(find rmader)/param/obstacle.yaml" command="load"/>
    </node>

    <node ns="$(arg quad)" name="rmader_commands" pkg="rmader" type="rmader_commands.py" output="screen"/>

    <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find rmader)/rqt_cfgs/demo_interface.perspective"/>

    <node pkg="behavior_selector" type="behavior_selector_node.py" name="behavior_selector" output="screen" />

</launch>